run_project_tests.py

```diff
@@ -393,6 +393,7 @@ def _run_test(testdir, test_build_dir, install_dir, extra_args, compiler, backen
 
 def gather_tests(testdir: Path):
     tests = [t.name for t in testdir.glob('*')]
+    tests = [t for t in tests if t != '.DS_Store'] # Filter non-tests files (macOS Finder indexing files, etc)
```

Comment:
I think you could just filter out all dotfiles here (since `Path.glob()` doesn't)

Reply:
Done 👍